.github/CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 <h3>New features since last release</h3>
 
+* A new hardware-efficient particle-conserving template has been implemented
```

Comment:
I think it's best to combine both templates in a single bullet point for the change log. Otherwise its too repetitive